Sources/OpenPass/OpenPassClient.swift

```diff
@@ -147,3 +147,46 @@ internal final class OpenPassClient {
         return try decoder.decode(ResponseType.self, from: data)
     }
 }
+
+extension OpenPassClient {
```

Comment:
Out of interest, why is this an extension function rather than just part of the client?